package.json

```diff
@@ -72,6 +72,7 @@
     "@sphereon/ssi-sdk.issuance-branding": "0.30.2-next.191",
     "@sphereon/ssi-sdk.mdl-mdoc": "0.30.2-next.191",
     "@sphereon/ssi-sdk.oid4vci-holder": "0.30.2-next.191",
+    "@sphereon/ssi-sdk.qr-code-generator": "^0.30.1",
```

Comment:
should have the same version as other ssi-sdk packages to avoid any issues

Reply:
ive update the version of the package to `0.30.2-next.191`